keyboards/uranuma/keymaps/default/keymap.c

```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+enum layer_number {
+  _BASE,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  BASE = SAFE_RANGE,
+  KANJI,
+  LOWER,
+  RAISE,
+};
+
+// Layer Mode aliases
+#define KC_LOWER LOWER
+#define KC_RAISE RAISE
+#define KC_ADUJST ADJUST
+#define KC_DLBS BASE //DF(_BASE)
+
+#define KC_RST   RESET
+#define KC_SUSFT SUSFT
+#define KC_ZSFT  LSFT_T(KC_Z)
+#define KC_MNSF  LSFT_T(KC_MINS)
+#define KC_ESCT  LCTL_T(KC_ESC)
+#define KC_TBAL  LALT_T(KC_TAB)
+#define KC_ALAL  LALT_T(KC_A)
+#define KC_ALAP  LALT_T(KC_APP)
+#define KC_JEQL  LSFT(KC_MINS)
+#define KC_WLCK  LALT(KC_L)
+#define KC_SFEQ  LSFT(KC_MINS)
+#define KC_SFPL  LSFT(KC_SCLN)
+#define KC_SFAS  LSFT(KC_QUOT)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT(
+  //|----------------------------------.                ,---------------------------------
+      KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,                    KC_Y, KC_U, KC_I, KC_O, KC_P,
+  //|------+------+------+------+------|                |------+------+------+------+-----
+      KC_A,  KC_S, KC_D,  KC_F, KC_G,                KC_H,  KC_J,  KC_K, KC_L,  KC_BSLS,
+  //|------+------+------+------+------|                |------+------+------+------+-----
+      KC_ZSFT, KC_X, KC_C, KC_V, KC_B,               KC_N, KC_M, KC_COMM, KC_DOT, KC_MNSF,
+  //|------+------+------+------+------|                |------+------+------+------+------
+      KC_ESCT, KC_TBAL, KC_PSCR, KC_LALT, KC_SPC, KC_ENT, LALT(KC_GRV), KC_SLSH, KC_BSLS, KC_RBRC,
+  //|------+------+------+------+------|                |------+------+------+------+------
+      KC_LSFT, MO(_LOWER),               KC_BSPC, KC_ENT,          MO(_RAISE),KC_RCTL
+  //|------+------+------+------+------|                |------+------+------+------+------
```

Comment:
Just a question here:

Would this be a reasonably accurate representation of this keymap for QMK Configurator?

![image](https://user-images.githubusercontent.com/18669334/76190931-65904400-619b-11ea-8e75-6a3d9b4827b1.png)

Reply:
Yes, you are right!